pyxrf/model/load_data_from_db.py

```diff
@@ -397,6 +467,10 @@ def map_data2D_srx(runid, fpath,
                             interp_list = (v.data[n][-1] - v.data[n][-3]) / 2 * np.arange(1, len_diff + 1) + v.data[n][-1]
                             data[n][m, min_len:datashape[1]] = interp_list
                 fluor_len = v.data['fluor'].shape[0]
+                if m > 0 and not (m % 10):
+                    print(f"Processed #{m} lines ...")
```

Comment:
Would it make sense to also add a total number of lines in the print?

Reply:
We discussed it in person. It's looping over a generator of events, so we don't directly know the length of it. Can be implemented in a separate PR later.